test/Generator/ParameterGeneratorTest.php

```diff
@@ -472,13 +473,19 @@ public function reflectionHintsProvider()
             [IterableHintsClass::class, 'iterableParameter', 'foo', 'iterable'],
             [IterableHintsClass::class, 'nullableIterableParameter', 'foo', '?iterable'],
             [IterableHintsClass::class, 'nullDefaultIterableParameter', 'foo', '?iterable'],
+            [ObjectHintsClass::class, 'objectParameter', 'foo', 'object'],
```

Comment:
These should only be added to the data providrr if the php version is 70200 or greater